audb/core/publish.py

```diff
@@ -175,6 +175,13 @@ def _find_media(
     def job(file):
         path = os.path.join(db_root, file)
         if file not in deps:
+            # enforce lowercase file extensions
```

Comment:
enforce sounds like we will change to lowercase

```suggestion
            # assert lowercase file extensions
```